src/mixins/segmentation/freehandPolylineRenderOverride.js

```diff
@@ -0,0 +1,42 @@
+import { draw, drawJoinedLines, getNewContext } from '../../drawing';
+import store from '../../store';
+
+const { getters } = store.modules.segmentation;
+
+/**
+ * Override for `freehandSegmentationMixin`'s `renderToolData` method to render a polyline instead
+ * of a freehand region with the first and last point connected. Apply after the `freehandSegmentationMixin`.
+ *
+ * @override
+ * @param {Object} evt The cornerstone render event.
+ * @returns {null}
+ */
+function renderToolData(evt) {
```

Comment:
Oh. Interesting. Not an alternative strategy _per se_, but an "alternative rendering strategy". We don't have a good way to switch here, except at time of class instantiation. Unless the tool itself provided both options in the render loop, and used `this.options` to determine rendering behavior.

Depending on intent, there may be a better way to surface. Regardless, I don't think `override` should be in the name. If mixins are general purpose, then you _could_ use this with a tool that doesn't have a render method specified? Maybe a name more specific to the kind of rendering it provides, so you know you're adding rendering behavior X?

Reply:
> then you could use this with a tool that doesn't have a render method specified?

You'd have none of the properties it refers to on `this` though.

Yeah I thought it was a pretty interesting approach that Romulo and Rodrigo came up with that reduced duplication, we can discuss.